U/util_linux/build_tarballs.jl

```diff
@@ -136,6 +144,7 @@ products = [
 dependencies = [
     # TOOD: verify Gettext is actually needed at runtime
     Dependency("Gettext_jll", v"0.20.1"; compat="=0.20.1"),
+    Dependency("SQLite_jll"),
```

Comment:
Looking at the upstream implementation, this is only needed for the program `lastlog2` (and its library), however we aren't listing `lastlog2` as an executable product.

Do we want to have that program, or can we disable building it and not need to add this dependency?